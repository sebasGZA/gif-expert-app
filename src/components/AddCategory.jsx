import { useState } from "react";

export const AddCategory = () => {
  const [inputValue, setInputValue] = useState("One Punch");

  const onInputChange = ({ target }) => {
    setInputValue(target.value);
  };

  const onSubmit = (event) => {
    event.preventDefault();
    console.log(inputValue);
  };

  return (
    <form onSubmit={onSubmit}>
      <input
        type="text"
        placeholder="Search gifs"
        value={inputValue}
        onChange={onInputChange}
      />
    </form>
  );
};
