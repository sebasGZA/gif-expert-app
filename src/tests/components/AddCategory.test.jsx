import { render } from "@testing-library/react";
import { AddCategory } from "../../components/AddCategory";

describe("Test in <AddCategory />", () => {
  test("Should make match with snapshot", () => {
    const { container } = render(<AddCategory />);

    expect(container).toMatchSnapshot();
  });
});
