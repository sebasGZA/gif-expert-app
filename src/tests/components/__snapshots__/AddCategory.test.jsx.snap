// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <AddCategory /> Should make match with snapshot 1`] = `
<div>
  <form>
    <input
      placeholder="Search gifs"
      type="text"
      value=""
    />
  </form>
</div>
`;
